src/codeu/chat/common/UserCollection.java

```diff
@@ -0,0 +1,16 @@
+package codeu.chat.common;
+
+import java.util.ArrayList;
+
+/**
+ * This class is simply to hold all of the users so it is easy to save them to a text file later
+ * Created by himonal on 6/13/17.
+ */
+public class UserCollection {
+    public User[] users;
+
+    public UserCollection(ArrayList<User> currentUsers) {
+        users = new User[currentUsers.size()];
+        users = currentUsers.toArray(users);
```

Comment:
Reading a little more through the code, I see you need a class definition to make use of the Gson library. So I'll let this slide. 

In general though, DRY (don't repeat yourself) is a really good rule of thumb to follow in software development, so your mental warning flags should go up when you see several nearly identical classes/segments of code.